src/software/networking/unix_sender.cpp

```diff
@@ -0,0 +1,47 @@
+#include "software/networking/unix_sender.h"
+
+#include <iostream>
+
+static unsigned MAX_SEND_FAILURES_BEFORE_LOG = 1000;
```

Comment:
nit: shouldn't this be in the header file?

Reply:
moved to private member